apps/website/src/components/calendar/Calendar.tsx

```diff
@@ -138,6 +143,36 @@ const getCalendarTheme = (month?: number): CalendarTheme => {
   };
 };
 
+export type MonthSelection = { month: number; year: number };
+
+export function useMonthSelection(initialDate?: Date) {
+  const [selected, setSelected] = useState<MonthSelection>();
+  useEffect(() => {
+    if (initialDate)
+      setSelected({
+        month: initialDate.getMonth(),
+        year: initialDate.getFullYear(),
+      });
+  }, [initialDate]);
+  return [selected, setSelected] as const;
+}
+
+export function useCalendarEventsQuery(selected?: MonthSelection) {
+  const start = useMemo(
+    () => selected && new Date(selected.year, selected.month, 1),
+    [selected],
+  );
+  const end = useMemo(
+    () => selected && new Date(selected.year, selected.month + 1, 1),
+    [selected],
+  );
```

Comment:
Do we need to consider timezone here? If an event was right at the end of the day Alveus time on the last day of the month, it wouldn't show in this query based on local time, I think?

Reply:
![image](https://github.com/user-attachments/assets/fb69c107-78df-4c59-88db-7870db31a453)
![image](https://github.com/user-attachments/assets/1aaa03ca-0e27-4feb-9cb1-f095f9968c47)
![image](https://github.com/user-attachments/assets/c1e29b1e-097b-4eca-b8e1-2ec9c128f389)
![image](https://github.com/user-attachments/assets/77486da8-c0bb-4518-8fc4-115b6cf4b616)

Reply:
(As an aside, the fading that is applied to previous days doesn't seem to be timezone-aware)